bundles/org.openhab.binding.kodi/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <thing:thing-descriptions bindingId="kodi"
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
-	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
```

Comment:
The schema should be url should start with `https://openhab.org` (see the github diff on this file. It should be what it was as shown in the github diff)

Reply:
I have modified the url back to openhab.org. 

PS: Will I use "tab" instead of "space", where I can see the rule of using space or using tab.